src/scripting/lua_map.cc

```diff
@@ -2442,13 +2490,15 @@ int LuaImmovableDescription::has_attribute(lua_State* L) {
 }
 
 /* RST
-   .. method:: probability_to_grow
+   .. method:: probability_to_grow(terrain_description)
 
-      :arg terrain: The terrain that we are checking the probability for.
-      :type terrain: :class:`wl.map.TerrainDescription`
+      (RO) For trees: Returns a double describing the probability that this tree will grow on the
+      given terrain. Returns nil if this tree has no terrain affinity (all trees should
+      have one).
```

Comment:
2× type markup

bushes also have terrain affinity

Reply:
Removed 'For Trees'